src/util.c

```diff
@@ -30,16 +30,24 @@ char *read_cmdline(void)
 	return NULL;
 }
 
-int hyper_setup_env(struct env *envs, int num)
+int hyper_setup_env(struct env *envs, int num, bool setPATH)
 {
 	int i, ret = 0;
 	struct env *env;
+	if (setPATH) {
+		ret = setenv("PATH", "/usr/local/bin:/usr/bin:/bin:.", 1);
```

Comment:
It's a good question, if the user is root, it's better to include them, otherwise `sbin` shouldn't be here. The problem is it is not easy to know who is the user in this position. That's part of what I said it is not as simple as I thought.